pmd-java/src/main/resources/rulesets/java/quickstart.xml

```diff
@@ -115,7 +115,7 @@
     <rule ref="category/java/codestyle.xml/UnnecessaryConstructor"/>
     <rule ref="category/java/codestyle.xml/UnnecessaryFullyQualifiedName"/>
     <rule ref="category/java/codestyle.xml/UnnecessaryImport" />
-    <rule ref="category/java/codestyle.xml/UnnecessaryLocalBeforeReturn"/>
+    <rule ref="category/java/codestyle.xml/VariableCanBeInlined"/>
```

Comment:
The list was sorted alphabetically before, did you notice?. Please move VariableCanBeInlined to the end (after line 123 after "UselessQualifiedThis").